.github/workflows/main.yml

```diff
@@ -26,5 +26,6 @@ jobs:
         pip install .
         pip install statsmodels cvxopt
         pip install git+https://github.com/jolars/pyslope.git
+        pip install lifelines
```

Comment:
add a comment: for Cox/
Easier for future maintenance to keep track of the need for each dependency 

thinking about it, in another PR these should go in the pyproject.toml in the test section